File: app/src/main/java/com/yonder/exercise/ui/favorites/IFavoritesRepository.java
package com.yonder.exercise.ui.favorites;

import android.arch.lifecycle.LiveData;

import com.yonder.exercise.db.BookModel;

import java.util.List;

/**
 * Created by YusufMac on 02/06/17.
 */

public interface IFavoritesRepository {
    LiveData<List<BookModel>> getFavoritedBooks();
}

File: app/src/main/java/com/yonder/exercise/ui/favorites/IFavoritesRepository.kt
package com.yonder.exercise.ui.favorites

import android.arch.lifecycle.LiveData

import com.yonder.exercise.db.BookModel

/**
 * Created by YusufMac on 02/06/17.
 */

interface IFavoritesRepository {
    val favoritedBooks: LiveData<List<BookModel>>
}
